bot/kodiak/tests/dependencies/test_dependencies.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
```

Comment:
should this test module be renamed to say "test_renovate.py` like the dependabot one?

Reply:
no, because this is general, testing both of them